### PR TITLE
Comprehensive Manual of Abhidhamma

### PR DESCRIPTION
**Pull request by JJM317**
Struggled with this one. Let me know what you think. I went with bodhi in the file link since he seems to be cited most. I also realize now that I saved it to canon, but I meant to put it in monograpghs. I won't change that until I hear from you. Thanks

### CODE REVIEW

File: _content/canon/comprehensive-manual-of-abhidhamma_bodhi.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Abhidhammattha-saṅgaha: A Comprehensive Manual of Abhidhamma"
+author: "Ācariya Anuruddha"
```
Comment (khemarato) [inline _content/canon/comprehensive-manual-of-abhidhamma_bodhi.md:3]: authors is a list

File: _content/canon/comprehensive-manual-of-abhidhamma_bodhi.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+editor:
```
Comment (khemarato) [inline _content/canon/comprehensive-manual-of-abhidhamma_bodhi.md:4]: editor singular (not a list)

File: _content/canon/comprehensive-manual-of-abhidhamma_bodhi.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Abhidhammattha-saṅgaha: A Comprehensive Manual of Abhidhamma"
+author: "Ācariya Anuruddha"
+editor: 
+- "bodhi"
+translator:
```
Comment (khemarato) [inline _content/canon/comprehensive-manual-of-abhidhamma_bodhi.md:6]: translator is also singular (not a list) Somehow you got these all backwards 🤭

File: _content/canon/comprehensive-manual-of-abhidhamma_bodhi.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+- "narada"
+external_url: "https://www.bps.lk/olib/bp/bp304s_Bfodhi_Comprehensive_Manual_of_Abhidhamma.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1ECzsN6iupxRPJBIXR20epwxT2dLMYG_m/view?usp=sharing"
+course: abhidhamma
+tags:
+  - ethics
```
Comment (khemarato) [inline _content/canon/comprehensive-manual-of-abhidhamma_bodhi.md:13]: Just `origination` I think? and none of the rest of these tags

**Review by khemarato — CHANGES_REQUESTED** [review]
Yeah. Canon is actually okay for this one, as it's a quasi-canonical Theravāda work. Monographs would work also, which do you prefer?

File: _content/canon/comprehensive-manual-of-abhidhamma_bodhi.md (hunk 1 above)
Comment (JJM317) [reply _content/canon/comprehensive-manual-of-abhidhamma_bodhi.md:3]: :)

**Review by khemarato — APPROVED** [review]

**Pull request merged**